request.go

```diff
@@ -32,8 +33,9 @@ type Request struct {
 	// append using RequestWithAfterDoFunc option
 	afterDoFuncs []func(req *Request, resp *Response) error
 
-	// response config
+	// retry config
 	maxAttempts int
+	bs          backoffStrategy
```

Comment:
Are you married to `bs` for a field name here? Seems a bit out of place with the other fields being somewhat verbose.

Reply:
haha, I had the same thought, but also liked using `bs` as a legitimate field name :)

Reply:
I'll change it to `backoffStrategy`